go/vt/vtgate/engine/dbddl.go

```diff
@@ -146,9 +150,9 @@ func (c *DBDDL) createDatabase(ctx context.Context, vcursor VCursor, plugin DBDD
 			if err != nil {
 				noErr = false
 				select {
-				case <-ctx.Done(): //context cancelled
+				case <-ctx.Done(): // context cancelled
 					return nil, vterrors.Errorf(vtrpc.Code_DEADLINE_EXCEEDED, "could not validate create database: tablets not healthy")
-				case <-time.After(500 * time.Millisecond): //timeout
+				case <-time.After(500 * time.Millisecond): // timeout
```

Comment:
feels weird to have `500 * time.Millisecond` copied in multiple places on this file. we should extract into a const/var